### ISSUE TITLE
Feature: Log response

### ISSUE THREAD

**Issue by DanieliMi** [iMi-digital/laravel-request-logger#14]
It would be really nice to see the response that is sent for the request. Including the status code.

**Comment by amenk** [comment]
First of all, this plugin is called request logger :-)

But the idea is nice, especially concerning the status code.

As for logging the full response, this should be optional or capped to a limit, because responses sometimes can get really big. (requests can get big, too - but in common use cases I would say that most requests are much smaller than the responses)

**Comment by DanieliMi** [comment]
I think a config option to turn on response logging would be nice.

**Comment by amenk** [comment]
Yes, but maybe something more differentiated, for example to log the response only for certain status codes and/or for certain content types.

**Comment by DanieliMi** [comment]
The config could be something like
```
log_response_for => [
    'status_codes' => [422, 500],
    'content_types' => ['application/json', 'application/xml'],
],
```
Then you can filter both by status code and content_type. If you would like to log everything I'd say passing `['*']` could be a good option.
